index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <nav id="navbar">
+        <header>
+          <h1>JS Documentation</h1>
+        </header>
+        <div class="nav-wrapper">
+          <ul>
+            <li>
+              <a class="nav-link" href="#Introduction">Introduction </a>
+              <hr />
+            </li>
+            <li>
+              <a class="nav-link" href="#what_you_should_already_know"
+                >what you should already know</a
+              >
+              <hr />
+            </li>
+            <li>
+              <a class="nav-link" href="#JavaScript_and_Java"
+                >JavaScript and Java</a
+              >
+              <hr />
+            </li>
+            <li>
+              <a class="nav-link" href="#Hello_world">Hello world</a>
+              <hr />
+            </li>
+            <li><a class="nav-link" href="#Variables">Variables</a></li>
+          </ul>
+        </div>
+      </nav>
+      <main id="main-doc">
+        <section id="Introduction" class="main-section">
+          <h2>Introduction</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
```

Comment:
Please limit the width of this paragraph to less than 100 characters.
![fcc-test-suite-template-line-width-comment](https://user-images.githubusercontent.com/44057863/143867907-74e2d48c-68eb-44c0-bb3c-4ebad9e0a6e7.png)